.travis.yml

```diff
@@ -12,11 +12,12 @@ gemfile:
   - gemfiles/rails_4.gemfile
   - gemfiles/rails_5.gemfile
 script:
-- bundle exec rspec
-- bundle exec codeclimate-test-reporter
+  - bundle exec rspec
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: 2.4.0
       gemfile: gemfiles/rails_3.gemfile
-
+addons:
+  code_climate:
+    repo_token: b65d6447fdd48601786f688ef382bd56fd597b906943ff5c27d3d0fbf1bfc966
```

Comment:
not 100% sure this will work the same in travis, but i think you're missing the bash script that sends the codecov report to the pr (I've seen travis listed in the docs and i know it's supported but the exact instructions; i can't find). 
anyway, perhaps try running `curl -s https://codecov.io/bash | bash` and see if this adds the commit statuses